src/Atoms/FooterProfileBtn/FooterProfileBtnStyled.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.button`
```

Comment:
is button as an element important? maybe it could also be a div or sth else?

Reply:
It could be, but why not button? We are going to click on it.